be/src/storage/rowset/segment_v2/segment_writer.cpp

```diff
@@ -275,7 +271,6 @@ Status SegmentWriter::append_chunk(const vectorized::Chunk& chunk) {
         }
         ++_row_count;
     }
```

Comment:
Could the _mem_tracker be removed in header file?

Reply:
will removed in next pr